src/SupervisionSuccess/model/calcOutcomesProportions.js

```diff
@@ -0,0 +1,74 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+/**
+ * Function that calculates outcome proportions from the given inputs.
```

Comment:
Can we expand this a bit to say, "Function that calculates outcome proportions from the given inputs. This determines what proportion of a change in outcomes (for both population size and cost) is attributable to the modeled changes in revocations versus new admissions."

Reply:
added